content/_data/changelogs/weekly.yml

```
@@ -4617,6 +4617,30 @@
     # pull: 4042 ([JENKINS-57678] - Change getInstance() (deprecated) to get())
     # pull: 4061 ([JENKINS-49316] Create Item: OK button disabled until TAB pressed)
     # pull: 4079 (IntelliJ/Java: Duplicate code -- Refactored as ExtensionFinder.getClassFromIndex)
+- version: '2.184'
+  date: 2019-07-07
+  changes:
+    - type: rfe
+      pull: 3939
+      issue: 25046
+      message: |-
+        Drop all sessions when logging out, preventing errors related to headers being too large.
```

```suggestion
        Drop all sessions when logging out, preventing errors related to headers being too large.
        It will work properly only for the default session ID cookie names.
```

And link to https://github.com/jenkinsci/extras-executable-war#jetty-session-ids ?

---

Adjusted the message.

I don't think a link here is necessary or useful, this change is unrelated to the fact that session cookie names can be changed (in fact, only useful when you don't).

---

I just wanted to add a disclaimer that this change applies only to default configurations so that users of nondefault session IDs know that they are on their own here (and this is fine with me)

---

Well, they won't have this problem in the first place, AFAIUI.